src/develop/masks.h

```diff
@@ -275,6 +275,8 @@ int dt_masks_legacy_params(dt_develop_t *dev, void *params, const int old_versio
 
 /** we create a completely new form. */
 dt_masks_form_t *dt_masks_create(dt_masks_type_t type);
+/** retrieve a form from a list of forms with is id */
```

Comment:
"with is id" -> "with its id"?

Same typo below, you're not the culprit here, but if you can fix it too.